packages/waku/src/cli.ts

```diff
@@ -96,6 +96,8 @@ async function runDev() {
   const config = await loadConfig();
   const app = new Hono();
   app.use('*', runner({ cmd: 'dev', config, env: process.env as any }));
+  // @ts-expect-error bypassing hono default notFound handler
+  app.use('*', (_c, _next) => {})
```

Comment:
```suggestion
  // bypassing hono default notFound handler
  // FIXME there might be a better way to handle this
  app.use('*', () => {})
```

Reply:
Without ts-ignore, this fails to pass typescript warnings during compilation. It's hard to read the hono middleware type signature to understand if there's another way that will pass type checking.

Reply:
Oh, I see. Then, just add FIXME comment.